cmd/ttn-lw-cli/commands/end_devices.go

```diff
@@ -1253,6 +1264,7 @@ func init() {
 	endDevicesCreateCommand.Flags().Bool("abp", false, "configure end device as ABP")
 	endDevicesCreateCommand.Flags().Bool("with-session", false, "generate ABP session DevAddr and keys")
 	endDevicesCreateCommand.Flags().Bool("with-claim-authentication-code", false, "generate claim authentication code of 4 bytes")
+	endDevicesCreateCommand.Flags().Bool("request-dev-eui", false, "request a new DevEUI from the configured address block")
```

Comment:
```suggestion
	endDevicesCreateCommand.Flags().Bool("request-dev-eui", false, "request a new DevEUI")
```

"the configured address block" is a bit confusing for the user as there's no configuration in the CLI for this.